README.md

```diff
@@ -1,102 +1,72 @@
-- [hello-storj](https://github.com/kaloyan-raev/hello-storj) - Example app based on android-libstorj (example, work-in-progress)
-- [SwiftyStorj](https://github.com/angu/SwiftyStorj) - Bindings to libstorj for iOS (example, work-in-progress)
-- [StorjDotNet](https://github.com/ssa3512/StorjDotNet) - .NET library for interacting with Storj (work-in-progress)
-- [libstorj.NET](https://github.com/TopperDEL/libstorj.NET) - Working .NET library based on java-libstorj, nuget-package available (work-in-progress)
-- [Storj.js](https://github.com/Storj/storj.js) - Browser library for interacting with Storj (work-in-progress)
-- [Storj Python Library](https://github.com/storj/storj-python-sdk) - Python library for interacting with Storj (work-in-progress)
-- [Storj-PHP](https://github.com/WebWeave/storj-php) - Implementation of the Storj protocol for PHP (work-in-progress)
+- **[Start here](https://www.storj.io/) - Register on satellites operated by Storj Labs**
+- [Documentation](https://docs.storj.io/dcs/) - Entrypoint to the official documentation
+- [Community support / forum](forum.storj.io/) - Community and support forum
+- [Storj Status](https://status.storj.io/) - Status of the Storj network
+- [Usage statistic dashboard](https://storjstats.info/) - Statistics about the full Storj network
+- [Whitepaper v3](https://storj.io/storj.pdf) - Actual version of Storj white-paper 
+- [FAQ](https://docs.storj.io/dcs/support/faqs) - Frequently Asked questions and answers 
+
+ ## Clients with Storj support and integrations 
+
+* **[uplink](https://github.com/storj/storj/releases) - Uplink is the official CLI for Storj. Can be downloaded from the release page** 
+* [Storj Mobile](https://play.google.com/store/apps/details?id=com.storj_mobile) (beta) - Android application to access Storj ([forum post](https://forum.storj.io/t/storj-mobile-beta-for-android/15578))
+* [rclone](https://rclone.org/tardigrade/) - Tool to sync your files to cloud storage including Storj 
+* [restic]() - Restic is a modern backup program. Supports Storj via the rclone . [howto](https://docs.storj.io/dcs/how-tos/backup-with-restic), [docs](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#other-services-via-rclone)
+
+* [FileZilla](https://docs.storj.io/dcs/how-tos/set-up-filezilla-for-decentralized-file-transfer) - New protocol for Storj is being added to FileZilla. 
+* [NextCloud](https://github.com/storj-thirdparty/nextcloud-app) - Adds Storj external storage to NextCloud 
+* [Duplicati](https://www.duplicati.com/) - Free backup software to store encrypted backups online ([howto](https://docs.storj.io/dcs/how-tos/backup-with-duplicati))
+
+# Client Libraries & language bindings
+
+Storj can be used via the native interface (uplink CLI and uplink libraries) or as an S3 compatible backend with any S3 compatible tool. The following list focuses on the native protocol.
+
+- [uplink](https://github.com/storj/uplink) - Go library to use Storj
```

Comment:
Wondering if we should not redirect to either our go docs page for easier reading?

Reply:
Good point, but I am not sure if we have good generated documentation for all the bidings. We have it for uplink. I would add the documentation link to the end of the line (and later I can replace it with a small `badge`-link link.) What do you think?

Reply:
That sounds good to me!